object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,14 @@
+/* exported defaults */
+
+/*
+  iterate through source object keys
+  if that key not in target keys then add to it
+*/
+
+function defaults(target, source) {
+  for (let sourceKey of Object.keys(source)) {
```

Comment:
you could use `const` instead of `let` since the value of `sourceKey` never changes.

Also, you can simplify your code a bit here if you use a `for... in` loop
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in